sources/packages/backend/apps/db-migrations/src/sql/Queue/Create-queue-configurations.sql

```diff
@@ -0,0 +1,15 @@
+-- Create new table queue_configurations.
+CREATE TABLE IF NOT EXISTS queue_configurations(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
```

Comment:
I guess that we have been trying to avoid DB-reserved words. Just a suggestion but maybe it would be better to go for `queue_name`.